dist/angular-disable-all.js

```diff
@@ -11,6 +11,25 @@
      * This module represents a disable-all directive which allows you to disable any element in the DOM.
      */
     angular.module('disableAll', []);
+    
+    /**
+     * @ngdoc directive
+     * @name doNotDisable
+     * @restrict A
+     * @description
+     * This directive allows you to do not disable the element even if it's under the disableAll directive
+     */
+    angular.module('disableAll').directive('doNotDisable', doNotDisableDirective);
```

Comment:
what do you think about alternative name like "skip-disable" or "skip-disabling"?